<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="balance">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>БАЛАНС:</string>
   </property>
  </widget>
  <widget class="QLabel" name="balance_sum">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS UI Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="info_del_money">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Расходы</string>
   </property>
  </widget>
  <widget class="QLabel" name="inf_add_money">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Доходы</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum_del_money">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="sum_add_money">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="information_mission">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>229</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;
background-color: #DCDCDC</string>
   </property>
   <property name="text">
    <string>Подробная информация</string>
   </property>
  </widget>
  <widget class="QLabel" name="mission">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ВАША ЦЕЛЬ:</string>
   </property>
  </widget>
  <widget class="QLabel" name="mission_sum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>368</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0/3900</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_mission">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QPushButton" name="mission_info">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;
background-color: #DCDCDC</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_money">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;
background-color: #DCDCDC</string>
   </property>
   <property name="text">
    <string>Добавить расход</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_money">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid; 
border-radius: 10px;
background-color: #DCDCDC</string>
   </property>
   <property name="text">
    <string>Добавить доход</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
